Recommendation/Item_content.py

```diff
@@ -67,7 +73,7 @@ def fetchDB(self,trails_col):
         print("Error Occured: Cause 'No Input'")
         raise

```

Comment:
Can you raise the exception like this:

`raise Exception("Error Occured: Cause 'No Input'")`